buttons/src/main/res/values/styles-icon-buttons.xml

```diff
@@ -0,0 +1,158 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <style name="Widget.Vitamin.IconButton.Primary.Medium" parent="@style/Widget.MaterialComponents.Button.UnelevatedButton">
+        <item name="android:layout_width">@dimen/vtmn_icon_buttons_size_medium</item>
```

Comment:
An attribute is declared for the kotlin code but the dimension is used here.
Is it possible to have the same code everywhere? (only dimens or only attribute if it's mandatory)

Reply:
I can use the reference or the dimension. Let's take a decision about the reference and I'll update this line according to the final result.

Reply:
Totally agree